src/index.html

```diff
@@ -5,8 +5,463 @@
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+              class="nav__link" 
+              data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">
+              Photo
+            </a>
+          </li>
+          
+          <li class="nav__item">
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalogue">
+      <ul class="catalogue__list">
+        <li class="catalogue__list-item">
+          <div class="card" data-qa="card">
+            <div class="card__image">
+              <img 
+            src="./images/imac.jpeg" 
+            alt="Apple A1419 iMac 27"
+            class="card__pic"
+            >
+            </div>
+            
+            <p class="card__description">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+      
+            <p class="card__code">
+              Product code: 195434
+            </p>
+      
+            <div class="card__rating">
+              <div class="stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+      
+              <a href="#home" class="card__reviews">
+                Reviews: 5
+              </a>
+            </div>
+      
+            <div class="card__prices">
+              <span class="card__price">Price:</span>
+      
+              <div class="card__amount">$2,199</div>
+            </div>
+      
+            <div class="card__buy">
+              <a 
+                href="#buy" 
+                class="card__buy-link"
+                data-qa="card-hover"
+              >BUY</a>
```

Comment:
format it properly + let's make it uppercase with styles
```suggestion
              <a 
                href="#buy" 
                class="card__buy-link"
                data-qa="card-hover"
              >
                buy
              </a>
```